### ISSUE TITLE
bad world seed: 949

### ISSUE THREAD

**Issue by mdII4bq3gIIa** [nroutasuo/level13#41]
Playing version 0.2.1 (alpha) and was able to wander off of the map and get stuck. On reload, I get dozens of console messages like this:

WARN: Level already contains sector 18.5.4 18.5.4
WARN: Level already contains sector 16.1.-5 16.1.-5
WARN: Level already contains sector 16.1.5 16.1.5

and this:

skip blocker at 13.-5.-1
skip blocker at 13.1.8
skip blocker at 13.0.10

At some point I was on my home square and the map allowed me to travel NW, even though there is no exit in that direction.

**Comment by nroutasuo** [comment]
The messages are actually fairly normal. What do you mean by wander off the map and get stuck? Did the buttons allow you to move into directions that don't exist?

**Comment by mdII4bq3gIIa** [comment]
Yes, I was able to travel off the map. It went something like this:

From my camp, I headed south (the only active exit) and then I think southeast, which I had not scouted before. I scouted that square, then went west and scouted that square. Then I returned two squares north to the camp. (This may not be the exact path but there are only so many ways to travel on this map.) I noticed the NW and another exit were active in addition to the south exit, even though there were not locations for those on the map. I had already entered the camp before I realized what I had seen on the map.

I thought maybe there were some secret passages, so after resting and collecting more supplies I returned to the map. The extra exits were still there. I went NW and was stuck. There was some sort of console message about a "location not found". After reloading I was back on the camp square.

I hope this is useful in some way.

**Comment by nroutasuo** [comment]
Yes, that's very helpful. Thanks for the details!

**Comment by mdII4bq3gIIa** [comment]
Seed 680 also appears to have a similar problem. After scouting the starting square, there is only one path leading south, but the NW and N exits show up as active in the navigation. But after reloading the page, those exits are not active.
